packages/react-component-library/src/components/CheckboxEnhanced/CheckboxEnhanced.test.tsx

```diff
@@ -99,7 +98,7 @@ describe('CheckboxEnhanced', () => {
     })
   })
 
-  describe('when a field has an error and the form has not been touched', () => {
+  describe.skip('when a field has an error and the form has not been touched', () => {
```

Comment:
The components with test skips are deprecated and are about to be removed from the code base.